cmd/guacone/cmd/vulnerability.go

```diff
@@ -606,41 +606,41 @@ func searchPkgViaHasSBOM(ctx context.Context, gqlclient graphql.Client, searchSt
 }
 
-func findDepPkgVersionIDs(ctx context.Context, gqlclient graphql.Client, depPkgType string, depPkgNameSpace string, depPkgName string, versionRange string) ([]string, error) {
-	var matchingDepPkgVersionIDs []string
-
-	depPkgFilter := &model.PkgSpec{
-		Type:      &depPkgType,
-		Namespace: &depPkgNameSpace,
-		Name:      &depPkgName,
-	}
-
-	depPkgResponse, err := model.Packages(ctx, gqlclient, *depPkgFilter)
-	if err != nil {
-		return nil, fmt.Errorf("error querying for dependent package: %w", err)
-	}
-
-	depPkgVersionsMap := map[string]string{}
-	depPkgVersions := []string{}
-	for _, depPkgVersion := range depPkgResponse.Packages[0].Namespaces[0].Names[0].Versions {
-		depPkgVersions = append(depPkgVersions, depPkgVersion.Version)
-		depPkgVersionsMap[depPkgVersion.Version] = depPkgVersion.Id
-	}
-
-	matchingDepPkgVersions, err := depversion.WhichVersionMatches(depPkgVersions, versionRange)
-	if err != nil {
-		// TODO(jeffmendoza): depversion is not handling all/new possible
-		// version ranges from deps.dev. Continue here to report possible
-		// vulns even if some paths cannot be followed.
-		matchingDepPkgVersions = nil
-		//return nil, nil, fmt.Errorf("error determining dependent version matches: %w", err)
-	}
-
-	for matchingDepPkgVersion := range matchingDepPkgVersions {
-		matchingDepPkgVersionIDs = append(matchingDepPkgVersionIDs, depPkgVersionsMap[matchingDepPkgVersion])
-	}
-	return matchingDepPkgVersionIDs, nil
-}
+//func FindDepPkgVersionIDs(ctx context.Context, gqlclient graphql.Client, depPkgType string, depPkgNameSpace string, depPkgName string, versionRange string) ([]string, error) {
+//	var matchingDepPkgVersionIDs []string
+//
+//	depPkgFilter := &model.PkgSpec{
+//		Type:      &depPkgType,
+//		Namespace: &depPkgNameSpace,
+//		Name:      &depPkgName,
+//	}
+//
+//	depPkgResponse, err := model.Packages(ctx, gqlclient, *depPkgFilter)
+//	if err != nil {
+//		return nil, fmt.Errorf("error querying for dependent package: %w", err)
+//	}
+//
+//	depPkgVersionsMap := map[string]string{}
+//	depPkgVersions := []string{}
+//	for _, depPkgVersion := range depPkgResponse.Packages[0].Namespaces[0].Names[0].Versions {
+//		depPkgVersions = append(depPkgVersions, depPkgVersion.Version)
+//		depPkgVersionsMap[depPkgVersion.Version] = depPkgVersion.Id
+//	}
+//
+//	matchingDepPkgVersions, err := depversion.WhichVersionMatches(depPkgVersions, versionRange)
+//	if err != nil {
+//		// TODO(jeffmendoza): depversion is not handling all/new possible
+//		// version ranges from deps.dev. Continue here to report possible
+//		// vulns even if some paths cannot be followed.
+//		matchingDepPkgVersions = nil
+//		//return nil, nil, fmt.Errorf("error determining dependent version matches: %w", err)
+//	}
+//
+//	for matchingDepPkgVersion := range matchingDepPkgVersions {
+//		matchingDepPkgVersionIDs = append(matchingDepPkgVersionIDs, depPkgVersionsMap[matchingDepPkgVersion])
+//	}
+//	return matchingDepPkgVersionIDs, nil
+//}
```

Comment:
any reason we are keeping the old code around?

Reply:
Oh, sorry, that was my mistake; I commented it out but forgot to delete it. Thanks!